README.md

```diff
@@ -1,71 +1,88 @@
 # Introduction
-The federation registry provides a secure web interface through which service operators can register and manage their OpenID Connect and Saml based service. A federation registry instance can serve multiple tenants and supports different AAI proxy technologies([SimpleSAMLphp](https://simplesamlphp.org/), [MITREid Connect](https://github.com/mitreid-connect/), SATOSA, Keycloak). Services can be managed through the portal and are deployed by sending configuration messages to [deployment agents](https://github.com/rciam/rciam-federation-registry-agent) that run in parallel with this project. Messages are exchanged using [Argo Messenging Service](https://grnet.gr/en/services/computing-and-storage-services/argo-messaging-service/).
 
-Federation registry is consisted of three node.js projects
-* federation-registry-frontend: A front-end portal created with Reactjs
-* federation-registry-backend-api: An API created with Expressjs
-* federation-registry-backend-ams-agent: An agent that uses the api and communicates with ams
+The federation registry provides a secure web interface through which service operators can register and manage their
+OpenID Connect and SAML based service. A federation registry instance can serve multiple tenants and supports different
+AAI proxy technologies([SimpleSAMLphp](https://simplesamlphp.org/), [MITREid Connect](https://github.com/mitreid-connect/),
+[SATOSA](https://daasi.de/en/satosa-a-modular-proxy/), [Keycloak](https://www.keycloak.org/)). Services can be managed
```

Comment:
I'd suggest to reorder based on community/popularity, e.g. "Keycloak, SimpleSAMLphp, SATOSA, MITREid Connect"

If there is no support for SATOSA yet, remove reference or add "planned"?